auto_rx/autorx/templates/index.html

```diff
@@ -1272,7 +1272,7 @@
                     document.getElementById("main").style.marginLeft = "350px";
                     document.getElementById("mySidenav").style.borderRadius = "0px 25px 25px 0px";
                     mymap.invalidateSize();
-                    setTimeout(scan_chart_obj.resize,500);
+                    scan_chart_obj.flush();

```

Comment:
Calling `resize()` here would blow away the size set in the constructor, so I've replaced the call with `flush()`, which forces a redraw. The 500-ms delay seems to have been unnecessary.